FirebaseInAppMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2021-12 -- v8.12.0
+- [fixed] Duplicate messages can occur when two campaigns are triggered by different events in In-App Messaging (#9070).
```

Comment:
Move to 8.11.0 section. 8.11.0 won't release until next month.

Reply:
Updated the year and month to current. Please lmk if it looks good.

Reply:
It should be 8.11.0 if it merges before the codefreeze. See go/firetracker

Reply:
Moved under 8.11.0. Please review.